helm-charts/falcon-integration-gateway/templates/configmap-env.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "falcon-integration-gateway.fullname" . }}-env
+  namespace: {{ .Release.Namespace }}
+  labels:
+     {{- include "falcon-integration-gateway.labels" . | nindent 4 }}
+data:
+  FALCON_CLOUD_REGION: {{ .Values.falcon.cloud_region | quote }}
+{{- if .Values.push.aws_security_hub.enabled }}
+  AWS_REGION: {{ .Values.push.aws_security_hub.region | quote }}
+{{- end }}
+{{- if .Values.push.azure_log_analytics.enabled }}
+  WORKSPACE_ID: {{ .Values.push.azure_log_analytics.workspace_id | quote }}
+  PRIMARY_KEY: {{ .Values.push.azure_log_analytics.primary_key | quote }}
+  ARC_AUTODISCOVERY: {{ .Values.push.azure_log_analytics.arc_autodiscovery | toString | quote }}
+{{- end }}
+{{- if .Values.push.chronicle.enabled }}
+  CHRONICLE_REGION: {{ .Values.push.chronicle.region | quote }}
+  GOOGLE_SECURITY_KEY: {{ .Values.push.chronicle.security_key | quote }}
+{{- end }}
+{{- if .Values.push.cloudtrail_lake.enabled }}
+  CLOUDTRAIL_LAKE_CHANNEL_ARN: {{ .Values.push.cloudtrail_lake.channel_arn | quote }}
+  CLOUDTRAIL_LAKE_REGION: {{ .Values.push.cloudtrail_lake.region | quote }}
+{{- end }}
+{{- if .Values.push.vmware_workspace_one.enabled }}
+  SYSLOG_HOST: {{ .Values.push.vmware_workspace_one.syslog_host | quote }}
+  SYSLOG_PORT: {{ .Values.push.vmware_workspace_one.syslog_port }}
+  WORKSPACEONE_TOKEN: {{ .Values.push.vmware_workspace_one.token | quote }}
+{{- end }}
```

Comment:
This should stay where it originally was in a secret since it still contains sensitive data. So please drop this configmap addition as well as the configmap file rename.

Reply:
The intent is to move the non-sensitive data out of the secret and into a config map where it belongs. If I set up an existing secret I don't want to be managing `SYSLOG_HOST` there for example. It should only contain sensitive data so I can keep the config managed in helm.

Another option is to use an `env:` key values

It looks like missed `GOOGLE_SECURITY_KEY` and WORKSPACEONE_TOKEN however. Can you clarify if anything else is sensitive?

Reply:
You'd want to update the config.ini and not create a new configmap. Technically, secrets are not secrets in k8s, and the use of `secretRef` should be avoided due to secrets as env var exposure, credential dumping, and exporting plaintext via logs. There are other sensitive values like `WORKSPACE_ID` and `PRIMARY_KEY`

Reply:
See changes

Reply:
The existing chart uses secretRef I just made the name here configurable.

In order to reduce scope of changes I moved to use `env` instead of attempting to figure out what was correctly a secret or not.  This allows me to set AWS_REGION in the vars as well as the other missing env vars.